lib/authorization/assertion/twitter_auth.rb

```diff
@@ -0,0 +1,49 @@
+module Authorization
+  module Assertion
+    class TwitterAuth
+      def initialize(access_token, access_token_secret)
+        @access_token = access_token
+        @access_token_secret = access_token_secret
+        @twitter_client = twitter_client
+        @user_data = user_data
```

Comment:
`@user_data` _probably_ doesn't need to be saved to an instance variable, I suspect Twitter will handle the caching in the client